### ISSUE TITLE
Breaking Out OpenContent Base from Accelerators Phase 1

### ISSUE THREAD

**Issue by aviking** [tsgrp/HPI#2263]
Creating a new OpenContent "project" which will contain the base OpenContent functionality needed to run AEV, Claim ACA and PnP ACA on Alfresco 6.x. The structure will be to build this new "project" which will create the OpenContent Product AMP and then deploy different Accelerators or AEV on top of it.

**Comment by aviking** [comment]
Initial commit of the OC Base "Project" commit with OC Revision: 26057
CR: dgrumieaux, ballen, gsteimer

Will close out this issue once documentation is complete.

**Comment by gsteimer** [comment]
This phase 1 effort should also include:

- [ ] Ensuring that multiple accelerators can be installed together (since we'll need this in our demo envs)
  - Minus the configs.  For this phase, we only need to ensure that the server components play nicely together and do not clash.  Having the ability to have accelerator specific configs can come in a later phase.
- [ ] Creating a "Demo" Accelerator that will be used for our demo environments
  - This would include all the non-accelerator types and 5. modules for engineering, contract management, etc etc
- [ ] Using the "base" OC plus the Claims, PnP and Demo accelerators on edge2 & release2
- [ ] Ensure that the PaaS release artifacts are the same as on-prem

**Comment by gsteimer** [comment]
Moved to Jira: https://alfresco.atlassian.net/browse/SEP-44
